### PR TITLE
Action/create

### PR DESCRIPTION
**Pull request by JosiahRegencia**

### CODE REVIEW

**Comment by greenmoss** [comment]
This is for #49 right?

File: savant/app/results.py [hunk 1]
```diff
@@ -25,11 +27,26 @@ def compare():
     get_scanned.get_items(filenames)
     record.record_comparison_flavors()
 
+    json_file = "Comparison-" + now.strftime("%Y-%m-%d_%H:%M") + ".json"
+
+    new_packages =  json.dumps({"Debs" : {"New" : comparison.new_debs()}},
```
Comment (greenmoss) [inline savant/app/results.py]: This failed the Code Climate test; can you refactor this as a single function? See https://codeclimate.com/github/Doveps/mono/pull/51

**Review by greenmoss — CHANGES_REQUESTED** [review]

**Comment by JosiahRegencia** [comment]
Yes. This is #49

**Comment by JosiahRegencia** [comment]
made changes regarding the code duplication

**Review by greenmoss — APPROVED** [review]

**Pull request merged**